Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/main/GameCardView.java

```diff
@@ -0,0 +1,98 @@
+package org.dolphinemu.dolphinemu.ui.main;
+
+import android.content.Context;
+import android.os.Parcel;
+import android.os.Parcelable;
+import android.support.v17.leanback.widget.ImageCardView;
+
+import org.dolphinemu.dolphinemu.R;
+
+/** View for showing a single game, meant for the TV interface. */
+public class GameCardView extends ImageCardView
```

Comment:
This class looks unneeded at all, I didn't test this but I think the old implementation was wrong and this PR followed the same path.

Why we don't use a simple xml based layout for the ImageCardView inside GameRowPresenter.onCreateViewHolder() and just use a simple color selector for the infoAreaBackgroundColor? do we really need to the custom class, shouldn't a simple color selector be able to switch the colors based on the selected state?

Another way could be to use a custom style as mentioned here:
https://developer.android.com/reference/android/support/v17/leanback/widget/ImageCardView.html

Looks like the custom style has the ability to change the infoBackgroundColor
https://chromium.googlesource.com/android_tools/+/882ee91503f28d1fd60ba939c5b37db6b9d4ad29/sdk/extras/android/support/v17/leanback/res/values/styles.xml#87

Reply:
That would be a lot better to use a selector and not having a custom class.

But I don't think any part of ImageCardView supports a ColorStateList though. infoAreaBackgroundColor is just a Color, even through styles, and I couldn't get View.setBackgroundTintMode to work with the ImageCardView.

Reply:
I just tested setInfoAreaBackground() with drawable selector and it worked holder.cardParent.setInfoAreaBackground(ContextCompat.getDrawable(holder.cardParent.getContext(), R.drawable.game_cube_selector));

not sure about the View.setBackgroundTintMode() I don't see where you used this in the PR, or it something you wanted to use?

Reply:
Ah, that makes sense. I didn't try to use drawable. I've updated to use that, please take a look.

Tint mode was just another thing I tried, but didn't work.